src/resmom/mom_main.c

```diff
@@ -110,7 +106,8 @@
 #include	"libsec.h"
 #include	"pbs_ecl.h"
 #include	"pbs_internal.h"
-#ifndef	WIN32
+#include	"avltree.h"
```

Comment:
same as above, now we don't use avltree.h in application logic directly, it should be pbs_idx.h